src/recipe/recipe.service.ts

```diff
@@ -20,6 +20,21 @@ export class RecipeService {
     private readonly s3Service: S3Service,
   ) {}
 
+  getImageUrlsToUrls(recipe: Recipe): Promise<string[]> {
```

Comment:
"image urls to urls"? Sounds strange to me. Should it be perhaps image keys to urls?

Reply:
I would rename it to `getPresignedUrlsforRecipeImages` or something like that